pkg/platform/packet/packet_test.go

```diff
@@ -355,3 +357,101 @@ func TestTerraformAddDeps(t *testing.T) {
 		})
 	}
 }
+
+func Test_findRepeatingString(t *testing.T) {
+	tests := []struct {
+		name string
+		strs []string
+		want string
+	}{
+		{
+			name: "repeated_strings",
+			strs: []string{"1", "2", "3", "1"},
+			want: "1",
+		},
+		{
+			name: "unique_strings",
+			strs: []string{"1", "2", "3"},
+			want: "",
+		},
+		{
+			name: "empty_input",
+			strs: []string{},
+			want: "",
+		},
+	}
+	for _, tt := range tests {
+		tt := tt
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			if got := findRepeatingString(tt.strs); got != tt.want {
+				t.Errorf("findRepeatingString() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func Test_validateNodePrivateCIDRs(t *testing.T) { //nolint:funlen
```

Comment:
Why don't we use the test grid we have set up above in `TestCheckValidConfig`?

Reply:
Yes, I'd simply add these test cases to TestCheckValidConfig

Reply:
We can't use `TestCheckValidConfig` because we don't call that function from `checkValidConfig`.

Reply:
Not to mention `TestCheckValidConfig` is only designed to test if the function passes or fails. While `Test_validateNodePrivateCIDRs` tests the output value of the function call if the outcome is not errorsome.

Reply:
>We can't use TestCheckValidConfig because we don't call that function from checkValidConfig.

I think we should then, since it's doing the configuration validation...

Reply:
done.

Reply:
Sorry, somehow I thought I saw the test called `createdTerraformConfigFile()`.